src/main/java/com/example/cats/domain/Order.java

```diff
@@ -0,0 +1,36 @@
+package com.example.cats.domain;
+
+import jakarta.persistence.*;
+import lombok.Data;
+
+import java.time.Instant;
+import java.util.List;
+
+@Entity
+@Table(name = "orders")
+@Data
+public class Order {
```

Comment:
that's wrong, as we in DDD have 3 levels, dto, domains and entity. You merged the domain and entity. -3 please rewatch lectures